capitalize_decorator.rb

```diff
@@ -0,0 +1,6 @@
+require_relative './BaseDecorator'
+class CapitalizeDecorator < BaseDecorator
+  def correct_name
+    super().capitalize
```

Comment:
`Implement a method correct_name that capitalizes the output of @nameable.correct_name` Kindly implement this requirement the right way ( ie ``@nameable.correct_name.capitalize``)